addons/test_mail/models/test_mail_thread_models.py

```diff
@@ -10,3 +10,21 @@ class MailTestCC(models.Model):
     _inherit = ['mail.thread.cc']
 
     name = fields.Char()
+
+
+class MailTestTrackingValue(models.Model):
+    _name = 'mail.test.tracking.value'
+    _description = 'Test tracking values'
+    _inherit = ['mail.thread']
+
+    boolean_field = fields.Boolean('Boolean', tracking=True)
+    char_field = fields.Char('Char', tracking=True)
+    currency_id = fields.Many2one('res.currency')
+    date_field = fields.Date('Date', tracking=True)
+    datetime_field = fields.Datetime('Datetime', tracking=True)
+    float_field = fields.Float('Float', tracking=True)
+    integer_field = fields.Integer('Integer', tracking=True)
+    monetary_field = fields.Monetary('Monetary', tracking=True)
+    partner_id = fields.Many2one('res.partner', string='Partner', tracking=True)
```

Comment:
Why does he have a specific naming ? Ok for currency (specific behavior), but this one could be one2many_field, unless you want to test something specific ?

Reply:
Hello, thanks for your review. I did it to reflect the fact that this field is linked to the `res.partner` model. Other fields are generic, this one is not. Do you still think it would be worthing it to name it with a generic name ?

Reply:
Is there any specific behavior linked to it being a m2o to res.partner ?

Reply:
When a record is created, this field expects an id corresponding to an existing `res.partner` record. So I'd say so.

Reply:
Standard behavior then. Could be a m2o towards any existing model in db, so we could probably name it many2one_field_id .

Reply:
I think the question is about whether the choice of `res.partner` as target is actually important or is arbitrary (is tracking a partner different than tracking another target model?), and if it's not important then the suggestion is to make the field name generic `many2one_field` just to reflect its type, but like all the other fields in this model for which we also don't care what they represent (but it's fine to keep `res.partner` as target, or to avoid confusion use another fake model I guess).

Reply:
We don't have that much fake models (and defining a new one just for that is probably not useful). Using res.partner is a good example of classic m2o we track so why not.